src_c/polygon.c

```diff
@@ -468,6 +468,34 @@ pg_polygon_move(pgPolygonObject *self, PyObject *const *args, Py_ssize_t nargs)
     return tmp;
 }
 
+static PyObject *
+pg_polygon_as_segments(pgPolygonObject *self, PyObject *_null) {
+    double *verts = self->polygon.vertices;
+    Py_ssize_t vert_num = self->polygon.verts_num;
```

Comment:
I'd rename this one to verts_num
```suggestion
    Py_ssize_t verts_num = self->polygon.verts_num;
```